### ISSUE TITLE
Is it needed to set mask=False when use wav2vec to inferrence ?

### ISSUE THREAD

**Issue by XuLw** [pytorch/fairseq#3815]
## ❓ Questions and Help

I search around but I don't find any useful help.

I use wav2vec2 to do a ASR model,

when testing, we found that the test accuracy is unstable

I think when the input data and the model is stable, the result has to be stable, bue not.

so I seach a lot and finally when I set mask=False
` def forward(
        self, source, padding_mask=None, mask=False, features_only=False,
        mask_indices=None, mask_channel_indices=None,
        padding_count=None,
    )` 
at wav2vec2.py

finally the test acc is stable ?

but I don't know this setting is right ? 

should I set **### mask=True** when I am doing a inferrence ? 

please !

**Comment by Kick28** [comment]
bump 😃 
There's a little chance to get the answer for this, but I still do my shot.
I also have the same question. Honestly speaking, I'm using HF Transformers implementation, but noticed in the config of my checkpoint the next:
```json
"apply_spec_augment": true,
"mask_time_prob": 0.075
```
I used `facebook/wav2vec2-xlsr-53-espeak-cv-ft checkpoint` as a base checkpoint for my fine-tuning, and those values in this model config are the same. So should we still use time masking during fine-tuning and inference?
